articles/daas/daas-ref-pricing.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Pricing information for UKCloud Desktop as a Service
+description: Provides useful information about UKCloud Desktop as a Service pricing, including pricing examples
+services: daas
+author: shighmoor
+reviewer:
+lastreviewed: 28/09/2020
+toc_rootlink: Service Information
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Pricing information
+toc_fullpath: Service Information/daas-ref-pricing.md
+toc_mdlink: daas-ref-pricing.md
+---
+
+# Pricing information for UKCloud Desktop as a Service
+
+## Overview
+
+UKCloud Desktop as a Service can achieve an effective price from £24.30 per desktop per month or £12.15* per user per month for Session Hosting access (*excluding RDS hosting). Full pricing with all options, including licensing and connectivity, are available in the [UKCloud Pricing Guide](https://ukcloud.com/pricing-guide). Additional UKCloud commitment discounts apply for this service.
+
+UKCloud will engage with you to assess your current desktop landscape, this includes understanding the end user roles, applications requirements, performance expectations, end user devices, security requirements, and connectivity needs. Following this initial assessment UKCloud will be able to recommend the most suitable deployment model and recommend desktop and application sizing to meet your requirement.
```

Comment:
Following this initial assessment UKCloud would then recommend and support you through a proof of concept, followed by larger pilot of DaaS to ensure the best possible user experience and cost can be achieved

Reply:
Done